frontend/src/pages/schedule/invitation/[submissionId].tsx

```diff
@@ -0,0 +1,307 @@
+/** @jsxRuntime classic */
+
+/** @jsx jsx */
+import React, { useCallback, useEffect } from "react";
+import { FormattedMessage } from "react-intl";
+import { useFormState } from "react-use-form-state";
+import {
+  Flex,
+  Box,
+  jsx,
+  Heading,
+  Text,
+  Button,
+  Textarea,
+  Label,
+  Radio,
+} from "theme-ui";
+
+import { GetStaticPaths, GetStaticProps } from "next";
+import { useRouter } from "next/router";
+
+import { getApolloClient, addApolloState } from "~/apollo/client";
+import { Alert } from "~/components/alert";
+import { prefetchSharedQueries } from "~/helpers/prefetch";
+import { useCurrentLanguage } from "~/locale/context";
+import { Language } from "~/locale/languages";
+import {
+  ScheduleInvitationOption,
+  useGetScheduleInvitationQuery,
+  useUpdateScheduleInvitationMutation,
+} from "~/types";
+
+type SpeakerResponseForm = {
+  option: string;
+  notes: string;
+};
+
+const EXTRA_NOTES_OPTIONS = [
+  ScheduleInvitationOption.Maybe,
+  ScheduleInvitationOption.Reject,
+];
+
+const formatDateTime = (datetime: string, language: Language) => {
+  const d = new Date(datetime);
+
+  const formatter = new Intl.DateTimeFormat(language, {
+    year: "numeric",
+    month: "long",
+    day: "numeric",
+    hour: "numeric",
+    minute: "numeric",
+  });
+
+  return formatter.format(d);
+};
+
+const Invitation = () => {
+  const language = useCurrentLanguage();
+  const router = useRouter();
+  const submissionId = router.query.submissionId as string;
+  const { loading, data, error } = useGetScheduleInvitationQuery({
+    variables: {
+      submissionId,
+    },
+  });
+  const [
+    updateScheduleInvitation,
+    { loading: isSubmitting, data: submitAnswerData },
+  ] = useUpdateScheduleInvitationMutation();
+  const [formState, { radio, text }] = useFormState<SpeakerResponseForm>({
+    option: ScheduleInvitationOption.Confirm,
+    notes: "",
+  });
+
+  console.log("submissionId", submissionId);
```

Comment:
```suggestion
```

Reply:
Are you suggesting the changes yourself? 😅

Reply:
yes! so I don't miss anything when I am done with it 😊